### PR TITLE
chore: add missing setuptools to win cli tests

### PR DESCRIPTION
**Pull request by hamidzr**
## Description

why now? a pip version change? 

## Test Plan

tests pass
## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .circleci/real_config.yml [hunk 1]
```diff
@@ -118,6 +118,12 @@ commands:
     steps:
       - run: echo 'CIRCLE_WORKING_DIRECTORY="${CIRCLE_WORKING_DIRECTORY/#\~/$HOME}"' >> $BASH_ENV
 
+  python-report:
+    description: "Help with debugging Python and pip related issues."
+    steps:
+      # keep it a single line to avoid adding ci overhead. each step is about 5s.
+      - run: python --version; pip --version; pip list; sh -c "pip check || true"
```
Comment (JComins000) [inline .circleci/real_config.yml:125]: can you try using block scalar syntax with `>`?
https://yaml-multiline.info/

Comment (JComins000) [reply .circleci/real_config.yml:125]: like this
```
- run: |
    git add --all
    git commit -am "a commit message"
    git push
```

Comment (hamidzr) [reply .circleci/real_config.yml:125]: 👍🏼 and "single step" would be more accurate for the comment

Comment (JComins000) [reply .circleci/real_config.yml:125]: id be cool with removing the comment altogether. let the syntax serve as an example. either way is cool

**Review by JComins000 — APPROVED** [review]

**Pull request merged**